docs/convert.md

```diff
@@ -0,0 +1,114 @@
+# Converting notebooks
+
+Framework’s built-in `convert` command helps you convert an [Observable notebook](https://observablehq.com/documentation/notebooks/) to standard [Markdown](./markdown) for use with Observable Framework. To convert a notebook, you just need its URL; pass it to the `convert` command like so:
+
+```sh echo
+npm run observable convert <notebook-url>
+```
+
+<div class="note">
+
+The `convert` command currently only supports public notebooks. To convert a private notebook, you can (temporarily) make the notebook public unlisted by clicking **Share…** on the notebook and choosing **Can view (unlisted)** under **Public** access. Please upvote [#1578](https://github.com/observablehq/framework/issues/1578) if you are interested in support for converting private notebooks.
+
+</div>
+
+<div class="tip">
+
+You can pass multiple URLs to convert many notebooks simultaneously.
+
+</div>
+
+For example, to convert D3’s [_Zoomable sunburst_](https://observablehq.com/@d3/zoomable-sunburst) example:
+
+```sh echo
+npm run observable convert https://observablehq.com/@d3/zoomable-sunburst
+```
+
+This will output something like:
+
+<style type="text/css">
+
+.focus {
+  color: var(--theme-foreground-focus);
+}
+
+.invert {
+  background-color: var(--theme-foreground-alt);
+  color: var(--theme-background);
+}
+
+</style>
+
+<pre><code><span class="muted">┌</span>  <span class="invert"> observable convert </span>
+<span class="muted">│</span>
+<span class="green">◇</span>  Downloaded <b>zoomable-sunburst.md</b> <span class="muted">in 443ms</span>
+<span class="muted">│</span>
+<span class="green">◇</span>  Downloaded <b>flare-2.json</b> <span class="muted">in 288ms</span>
+<span class="muted">│</span>
+<span class="muted">└</span>  1 notebook converted; 2 files written</code></pre>
+
+The `convert` command generates files in the current working directory. The command above generates two files: <code>zoomable-sunburst.md</code>, a Markdown file representing the converted notebook; and <code>flare-2.json</code>, an attached JSON file.
```

Comment:
```suggestion
The `convert` command saves its output in the project folder. The command above generates two files: <code>zoomable-sunburst.md</code>, a Markdown file representing the converted notebook; and <code>flare-2.json</code>, an attached JSON file.
```

Is this a bug? The `--output=.` default option is _relative to the project's folder_ not to the cwd. So if framework is installed in `~/fil/example-project/` and I cd into `~/fil/example-project/src/here/`, then run `npm run observable convert https://observablehq.com/@d3/zoomable-sunburst`, the files are created in `~/fil/example-project/`.

Reply:
https://github.com/observablehq/framework/issues/838